milli/src/search/new/sort.rs

```diff
@@ -150,7 +183,15 @@ impl<'ctx, Query: RankingRuleQueryTrait> RankingRule<'ctx, Query> for Sort<'ctx,
             Ok(Some(bucket))
         } else {
             let query = self.original_query.as_ref().unwrap().clone();
-            Ok(Some(RankingRuleOutput { query, candidates: universe.clone() }))
+            Ok(Some(RankingRuleOutput {
+                query,
+                candidates: universe.clone(),
+                score: ScoreDetails::Sort(score_details::Sort {
+                    field_name: self.field_name.clone(),
+                    ascending: self.is_ascending,
+                    value: serde_json::Value::Null,
```

Comment:
hmm... I don't think this is technically correct. We could return documents whose value for this field is different than `null` (ie field doesn't exist, or contains an empty array, or maybe even an object). It doesn't necessarily make it wrong, but it is tricky.